ts/nni_manager/training_service/reusable/environments/amlEnvironmentService.ts

```diff
@@ -114,7 +117,7 @@ export class AMLEnvironmentService extends EnvironmentService {
         }
         const amlEnvironment: AMLEnvironmentInformation = environment as AMLEnvironmentInformation;
         const environmentLocalTempFolder = path.join(this.experimentRootDir, this.experimentId, "environment-temp");
-        environment.command = `import os\nos.system('${amlEnvironment.command}')`;
+        environment.command = `import os\nos.system('mv envs outputs/envs && cd outputs && ${amlEnvironment.command}')`;
```

Comment:
Will this work for windows?

Reply:
This command will be written in `nni_script.py`, and always launch in a container, seems we only support Linux based image?

Reply:
Yes, only support linux image.

Reply:
Is `output` folder created by NNI?

Reply:
yes, NNI will create `outputs` and `logs`